README.md

```diff
@@ -17,19 +17,11 @@
 * Service Wizard: [KIDL spec](https://github.com/kbase/service_wizard/blob/master/ServiceWizard.spec)
 * Catalog: [KIDL spec](https://github.com/kbase/catalog/blob/master/catalog.spec)
 
-### Relation Engine codebases
+### Relation Engine and Search codebases
 
-* HTTP & JSON API: [https://github.com/kbase/relation_engine_api](https://github.com/kbase/relation_engine_api)
-* Schemas and specifications for the API: [https://github.com/kbase/relation_engine_spec](https://github.com/kbase/relation_engine_spec)
-* Data import scripts: [https://github.com/kbase/relation_engine_importers](https://github.com/kbase/relation_engine_importers)
-* Workspace importer/loader/sync (via Kafka): [https://github.com/kbaseapps/relation_engine_sync](https://github.com/kbaseapps/relation_engine_sync)
-* Biochemistry reaction data: [https://github.com/kbaseapps/BiochemistryAPI](https://github.com/kbaseapps/BiochemistryAPI)
-
-### Search codebases
-
-* HTTP & JSON API for performing searches: [https://github.com/kbase/search_api_deluxe](https://github.com/kbase/search_api_deluxe)
-* Index creation and syncing from the workspace via Kafka: [https://github.com/kbase/index_runner_deluxe](https://github.com/kbase/index_runner_deluxe)
-* Global search configuration: [https://github.com/kbase/search_config](https://github.com/kbase/search_config)
+* The Relation Engine (using ArangoDB) can be found here: [https://github.com/kbase/relation_engine](https://github.com/kbase/relation_engine)
```

Comment:
Do you want to mention the importers repo, https://github.com/kbase/relation_engine_importers?

Reply:
I do want to migrate and deprecate that repo, but not sure how long until that happens. I think let's just leave it out